lib/Date/Holidays/Adapter/AW.pm

```diff
@@ -27,8 +29,12 @@ sub is_holiday {
 
     my $holiday;
 
+    my $y = delete $params{year};
+    my $m = delete $params{month};
+    my $d = delete $params{day};
+
```

Comment:
Please write these out for readability and consistency throughout the code.

Meaning `$year` over `$y` and so forth

Reply:
They have been updated.